test/test_distributed.py

```diff
@@ -1718,6 +1718,18 @@ def test_DistributedDataParallelCPU(self):
         )
         self._barrier()
 
+    @unittest.skipIf(BACKEND != 'nccl' and BACKEND != 'gloo',
+                     "Only Nccl & Gloo backend support DistributedDataParallel")
+    @skip_if_no_cuda_distributed
+    @skip_if_no_gpu
+    def test_DistributedDataParallel_python(self):
+        group, group_id, rank = self._init_global_test()
+        rank_to_GPU = self._init_multigpu_helper()
+        gpus = list(rank_to_GPU[rank])
+
+        # a module without gradients shouldn't be accepted
+        self.assertRaises(AssertionError, lambda: nn.parallel.DistributedDataParallel(nn.Module(), device_ids=gpus))
```

Comment:
Not strictly needed, but you can turn this into a test that doesn't need GPUs, since it doesn't allocate tensors.